### DISCUSSION TITLE
第一次来绳网，不知道发点啥，给大家分享一下我最近从六分街的影像店借来看的影片吧，名字叫《Girls band cry》，讲述空洞另一头的一个地区中有一位女孩因为在学校中遇到了不公待遇，索性离开家乡，然后载光映广场遇到了自己喜欢的偶像，于是退学开始搞乐队的故事，我想我一辈子也忘不了这个作品吧

### DISCUSSION THREAD

**Discussion by RaInWH** [share121/inter-knot#4787] [其他]
![Uploading 1725960833522.png…]()

**Comment by RaInWH** [comment]
![Uploading 1725960833522.png…]()

**Comment by RaInWH** [comment]
打错字了，在光映广场

**Comment by lllliufeng** [comment]
啊？<img src="https://upload-bbs.miyoushe.com/upload/2022/12/08/a2fa805a0a782cdb8ab44ddaf8d5343d_7408497524870727211.png" alt="阿姬倒地" width="75" >（图片没传完就发了吧）

**Reply by RaInWH** [reply]
好像是的吧(｡•ˇ‸ˇ•｡)
